### PR TITLE
Update sampling

### PR DESCRIPTION
**Pull request by jeremyrhyde**
Updating the endpoints

### CODE REVIEW

**Comment by nicksanford** [comment]
@jeremyrhyde is this ready to merge? If so can you please mark it as ready to review & ask me for a review?

File: sample.sh [hunk 1]
```diff
@@ -1,41 +1,30 @@
...
+  -plaintext -protoset <(~/viam-cartographer/grpc/bin/buf build -o -)  $GRPC_SERVER  viam.service.slam.v1.SLAMService/GetPosition > \
   ./sample/position/${DATE}.json 
 
 grpcurl -max-msg-sz 100000000000000 -d '{"name": "slam"}' \
-  -plaintext -protoset <(~/slam/slam-libraries/grpc/bin/buf build -o -)  $GRPC_SERVER  viam.service.slam.v1.SLAMService/GetPositionNew > \
-  ./sample/position_new/${DATE}.json 
-
-grpcurl -max-msg-sz 100000000000000 -d '{"name": "slam"}' \
-  -plaintext -protoset <(~/slam/slam-libraries/grpc/bin/buf build -o -) $GRPC_SERVER  viam.service.slam.v1.SLAMService.GetPointCloudMap \
+  -plaintext -protoset <(~/viam-cartographer/grpc/bin/buf build -o -) $GRPC_SERVER  viam.service.slam.v1.SLAMService.GetPointCloudMap \
```
Comment (nicksanford) [inline sample.sh]: @jeremyrhyde this doesn't work right? Don't you need the base64_decode.py I wrote for this to work?

File: sample.sh [hunk 2]
```diff
@@ -1,41 +1,30 @@
 #!/bin/bash
 
-DATE=$(date +"%Y-%m-%dT%H:%M%:%SZ")
+DATE=$(date +"%Y-%m-%dT%H:%M:%S")
 echo "Run ${DATE}"
 GRPC_SERVER="localhost:8083"
 # GRPC_SERVER=$(sudo lsof -i -n -P | grep carto | awk '{ print $9 }' | head -n 1)
-SLAM_DIR="${HOME}/slam"
-API_DIR="${HOME}/api"
+SLAM_DIR="${HOME}/Development/viam-cartographer"
```
Comment (jeremyrhyde) [inline sample.sh:7]: [note] I can revert these if desired

Comment (nicksanford) [reply sample.sh:7]: this is fine

**Review by nicksanford — APPROVED** [review]

**Pull request merged**